frontend/src/components/WorkoutTemplateModal/WorkoutTemplateModal.tsx

```diff
@@ -107,6 +114,31 @@ const WorkoutTemplateModal: React.FC<WorkoutTemplateModalProps> = ({
           placeholder="Workout template title"
           className={styles.input}
         />
+        <div className={styles.radioSection}>
+          <label className={styles.label}>Make template Public ?</label>
+          <div className={styles.optionsContainer}>
+            <label className={styles.option}>
+              <input
+                type="radio"
+                value="true"
```

Comment:
Is there a better way to do this instead of using two inputs?

Can you do `value={isPublic? 'true' : 'false'}`?

Reply:
I don't think I understand the approach you are talking about @m5matthew